src/jira_client.ts

```diff
@@ -0,0 +1,381 @@
+import { toError } from "fp-ts/lib/Either";
+import {
+  fromEither,
+  fromLeft,
+  taskEither,
+  TaskEither,
+  tryCatch
+} from "fp-ts/lib/TaskEither";
+import * as t from "io-ts";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import nodeFetch from "node-fetch";
+import { ServiceId } from "../generated/api/ServiceId";
+import * as config from "./config";
+
+export const JIRA_SERVICE_TAG_PREFIX = "devportal-service-";
+export const JIRA_DISABLE_LABEL = "DISATTIVAZIONE";
+
+const JIRA_CONFIG = config.getJiraConfigOrThrow();
+
+export const SearchJiraIssueResponse = t.interface({
```

Comment:
Otherwise we can generate the entire Jira Client from OpenApi specs using `openapi-codegen-ts`. This for sure on another PR even after the release on production.